nautobot_firewall_models/templates/nautobot_firewall_models/inc/natpolicyrule_tablehead.html

```diff
@@ -0,0 +1,26 @@
+<tr>
+    <th style="border: 1px solid #ccc; vertical-align: bottom;" rowspan=3>Index</th>
+    <th style="border: 1px solid #ccc; text-align: center;" colspan=6>Source</th>
```

Comment:
Thoughts on the top level table heads being?

* Index
* Original
* Translated
* Mode
* Log